pkg/kubelet/kubelet.go

```diff
@@ -3095,6 +3095,8 @@ func (kl *Kubelet) isContainerRuntimeVersionCompatible() error {
 		if result < 0 {
 			return fmt.Errorf("container runtime version is older than %s", dockertools.MinimumDockerAPIVersion)
 		}
+	case "rkt":
+		return kl.GetRuntime().(*rkt.Runtime).CheckVersion(rkt.MinimumRktBinVersion, rkt.RecommendedRktBinVersion, rkt.MinimumAppcVersion, rkt.MinimumRktApiVersion, rkt.MinimumSystemdVersion)
```

Comment:
Yes, the container runtime has enough information to decide whether the versions are compatible. We should push the implementation down to the runtime. This will make rkt's implementation easier too.